### PR TITLE
Add Isaiah

### PR DESCRIPTION
**Pull request by HisuianZoroark**

### CODE REVIEW

File: data/mods/ssb/abilities.ts [hunk 1]
```diff
@@ -198,6 +198,23 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
...
+				if (pokemon.volatiles['powertrick']) return;
+				this.add('-ability', pokemon, 'Anchor Arms');
+				pokemon.addVolatile('powertrick');
+			} else {
+				if (!pokemon.volatiles['powertrick']) return;
+				this.add('-ability', pokemon, 'Anchor Arms');
+				pokemon.removeVolatile('powertrick');
+			}
+		},
+	},
```
Comment (KrisXV) [inline data/mods/ssb/abilities.ts:216]:
```suggestion
		onStart(pokemon) {
			if (this.field.getPseudoWeather('trickroom') && !pokemon.volatiles['powertrick']) {
				this.add('-ability', pokemon, 'Anchor Arms');
				pokemon.addVolatile('powertrick');
			}
		},
		onAnyPseudoWeatherChange(pokemon, source, pseudoWeather) {
			if (this.field.getPseudoWeather('trickroom')) {
				if (pokemon.volatiles['powertrick']) return;
				this.add('-ability', pokemon, 'Anchor Arms');
				pokemon.addVolatile('powertrick');
			} else {
				if (!pokemon.volatiles['powertrick']) return;
				this.add('-ability', pokemon, 'Anchor Arms');
				pokemon.removeVolatile('powertrick');
			}
		},
	},
```

Comment (Karthik99999) [reply data/mods/ssb/abilities.ts:216]: `pokemon` should be `this.effectState.target`

Comment (KrisXV) [reply data/mods/ssb/abilities.ts:216]: ur just mad that im prettier than u

File: data/mods/ssb/abilities.ts [hunk 2]
```diff
@@ -231,51 +247,32 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
 	youllneverwalkalone: {
 		shortDesc: "Boosts Atk, Def, SpD, and Spe by 25% under Anfield Atmosphere.",
 		name: "You'll Never Walk Alone",
```
Comment (KrisXV) [inline data/mods/ssb/abilities.ts:249]:
```suggestion
		name: "You'll Never Walk Alone",
		onStart(pokemon) {
			if (this.field.getPseudoWeather('anfieldatmosphere')) {
				this.add('-ability', pokemon, 'You\'ll Never Walk Alone');
			}
		},
```
maybe if we could get a way to detect when it ended to say the boosts were removed? idk thats a task for later

**Pull request merged**